pypit/ardebug.py

```diff
@@ -17,3 +21,19 @@ def init():
                  trace_obj=False,
                  )
     return debug
+
+
+def show_image(args, **kwargs):
+    """ Wrapper to pypit_ginga
+    Parameters
+    ----------
+    args
+    kwargs
+
+    Returns
+    -------
+
+    """
+    return pyp_g.show_image(args, **kwargs)
+
+from pdb import *
```

Comment:
Is this line necessary? If so, should we instead do our standard `debugger` code?